hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -139,7 +139,6 @@ public static void init() throws Exception {
         .setOMServiceId(omServiceId)
         .setNumOfOzoneManagers(numOfOMs)
         .build();
-    conf.setQuietMode(false);
```

Comment:
the quiet mode was suppressed because Hadoop 3.3 expects hdfs-rbf-default.xml which we don't have.
Another workaround proposed by @smengcl is to add a dummy hdfs-rbf-default.xml as a resource file.

Reply:
BTW, discussed with @smengcl offline, it seems strange we load hdfs-client jar (which is the culprit of this test error). Do we really need hdfs-client in Ozone?

Reply:
@adoroszlai I thought we were intentionally placing `conf.setQuietMode(false)` here uncover any warnings in the tests?

Reply:
> it seems strange we load hdfs-client jar (which is the culprit of this test error)

OM uses `URLConnectionFactory` from this lib.

What I found strange was that other tests with `setQuietMode(false)` are passing.

Reply:
>What I found strange was that other tests with setQuietMode(false) are passing.

There is no FileSystem.get(conf) in those classes I think. The problem surfaces because of that only.

Reply:
@smengcl I don't understand the importance of keeping quiet mode disabled.  The config file is not really required as everything works in the default quiet mode.  What is the benefit of adding the empty config file?

I have merged the change to unblock follow-up work on protobuf upgrade.  If you have any concerns about the change to this single test, please file a new issue and we can discuss/address it there.

Reply:
@adoroszlai Ok. That works for me. Thx!